lib/generators/apple_auth/apple_sign_in_controller/templates/apple_sign_in_controller.rb

```diff
@@ -21,7 +21,8 @@ def apple_validate
       apple_sign_in_params[:user_identity],
       apple_sign_in_params[:jwt]
     ).validate!
-    AppleAuth::Token.new(apple_sign_in_params[:code]).authenticate
+    # next call will return a hash with access_token, refresh_token, expires_at
```

Comment:
I understand the purpose of this, but I think `reponse_hash` method is already descriptive enough, can we get rid of this comment?

Reply:
It was an idea to be more descriptive, in the case that the user wants to implement the refresh the token part and find easy where he will found it, but we can remove it